Package.swift

```diff
@@ -22,26 +31,42 @@ let package = Package(
             name: "WorkflowSwiftUI",
             targets: ["WorkflowSwiftUI"]
         ),
+
+        // MARK: WorkflowReactiveSwift
+
         .library(
             name: "WorkflowReactiveSwift",
             targets: ["WorkflowReactiveSwift"]
         ),
         .library(
-            name: "WorkflowCombine",
-            targets: ["WorkflowCombine"]
+            name: "WorkflowReactiveSwiftTesting",
+            targets: ["WorkflowReactiveSwiftTesting"]
         ),
+
+        // MARK: WorkflowRxSwift
+
         .library(
-            name: "WorkflowTesting",
-            targets: ["WorkflowTesting"]
+            name: "WorkflowRxSwift",
```

Comment:
Should this be `WorkflowReactiveSwift`?

Reply:
Whoops! it was the `target` that was the issue